src/components/layout/Header.js

```diff
@@ -24,8 +24,10 @@ const useStyles = makeStyles((theme) => ({
     alignItems: 'center',
   },
   logo: {
-    height: '40px',
-    margin: theme.spacing(0, 2),
+    fill: 'white',
+  },
+  title: {
+    margin: theme.spacing(0, 1.5),
```

Comment:
Not sure it's best practice to use decimals, as this then gets multiplied by a pixel number and could result in half pixels.

https://material-ui.com/customization/spacing/